controllers/manila_controller.go

```diff
@@ -521,6 +530,56 @@ func (r *ManilaReconciler) reconcileNormal(ctx context.Context, instance *manila
 
 	// end transportURL
 
+	//
+	// create NotificationBus transportURL CR and get the actual URL from the
+	// associated secret that is created
+	//
+
+	// Request TransportURL when the parameter is provided in the CR
+	// and it does not match with the existing RabbitMqClusterName
+	if instance.Spec.NotificationBusInstance != nil {
+
+		// init .Status.NotificationURLSecret
+		if instance.Status.NotificationURLSecret == nil {
+			instance.Status.NotificationURLSecret = ptr.To("")
+		}
```

Comment:
I guess we could remove this init if we just use this bellow? `		instance.Status.NotificationURLSecret = ptr.To(notificationBusInstanceURL.Status.SecretName)`